src/index.js

```diff
@@ -1689,13 +1690,28 @@ export class ULabel {
         }
     }
 
-    // Create a new canvas for an individual annotation and return its context
-    get_init_canvas_context_id(annotation_id, subtask = null) {
+    // Find the next available annotation context and return its ID
+    // If all contexts are in use, create a new one
+    get_next_available_canvas_id(subtask = null) {
```

Comment:
```suggestion
    /**
     * Find the next available annotation context and return its ID.
     * If all annotation contexts are in use, create a new canvas and return it's id.
     * @param {ULabelSubtask} subtask 
     * @returns {string} The ID of an available canvas.
     */
    get_next_available_canvas_id(subtask = null) {
```
I know that we haven't been very consistent with this, but we should move towards giving every function/method a docstring with a brief overview of its purpose. [Here](https://google.github.io/styleguide/jsguide.html#jsdoc) is documentation outlining JavaScript best practices as a whole, but section 7 is the section on docstrings.

Reply:
[9d64b2d](https://github.com/SenteraLLC/ulabel/pull/152/commits/9d64b2d52e57d44b18106b2f7784804d691cfc91)